### PR TITLE
First pass at WTD Vilnius

### PR DESCRIPTION
**Pull request by plaindocs**
Holding page only so far, set `flaglanding: True` to False and build locally to see the rest of the site.

There are assorted `TODO` and `TBC` which need fixing mostly in `docs/_data/config-vilnius-2019.yml` and `docs/conf/vilnius/2019`.

Possibly also in `docs/conf/include/`. 

- [x] ticket prices
- [ ] ticket links (next round)
- [ ] cfp (next round)
- [ ] First news item

### CODE REVIEW

**Comment by plaindocs** [comment]
@aidiss @thatdocslady preview here https://deploy-preview-722--dentist-canary-64055.netlify.com/conf/vilnius/2019/

**Comment by plaindocs** [comment]
- [x] OK we also need the sponsorship levels, also marked `#TODO` in the config file.

**Comment by plaindocs** [comment]
And maybe think about some more announcement text in `new/init.rst` as it is the first event, not just a reup of last year.

**Comment by thatdocslady** [comment]
I'm on the blog post copy, @aidiss please see Sam's other comments :)

**Comment by plaindocs** [comment]
Yep, we're getting there.

Will there be a job fair?

**Comment by plaindocs** [comment]
I'm guessing not?

**Comment by thatdocslady** [comment]
Nope, no unconference or job fair. Lightning talks TBD. It'll be writing day on Sunday and talks on Monday.

File: docs/_templates/2019/lt/menu-mobile.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                          {% endif %}
+                          <li><a href="{{ pathto('conf/vilnius/2019/schedule') }}">Schedule</a></li>
+                          <li><a href="{{ pathto('conf/vilnius/2019/speakers') }}">Talks</a></li>
+                          <li><a href="{{ pathto('conf/vilnius/2019/lightning-talks') }}">Lightning Talks</a></li>
+                          <li><a href="{{ pathto('conf/vilnius/2019/unconference') }}">Unconference</a></li>
+                          <li><a href="{{ pathto('conf/vilnius/2019/writing-day') }}">Writing Day</a></li>
+                          <li><a href="{{ pathto('conf/vilnius/2019/job-fair') }}">Job Fair</a></li>
+                          <li><a href="{{ pathto('conf/vilnius/2019/badge-flair') }}">Badge Flair Contest</a></li>
+                          {% if flaghashike %}
+                          <li><a href="{{ pathto('conf/portland/2019/outing') }}">Hike</a></li>
```
Comment (plaindocs) [inline docs/_templates/2019/lt/menu-mobile.html:20]:
```suggestion
                          <li><a href="{{ pathto('conf/vilnius/2019/outing') }}">Hike</a></li>
```

File: docs/_templates/2019/lt/menu-desktop.html [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+                                          <li><a href="{{ pathto('conf/vilnius/2019/schedule') }}">Schedule</a></li>
+                                          <li><a href="{{ pathto('conf/vilnius/2019/speakers') }}">Talks</a></li>
+                                          <li><a href="{{ pathto('conf/vilnius/2019/lightning-talks') }}">Lightning Talks</a></li>
+                                          <li><a href="{{ pathto('conf/vilnius/2019/unconference') }}">Unconference</a></li>
+                                          <li><a href="{{ pathto('conf/vilnius/2019/writing-day') }}">Writing Day</a></li>
+                                          <li><a href="{{ pathto('conf/vilnius/2019/job-fair') }}">Job Fair</a></li>
+                                          <li><a href="{{ pathto('conf/vilnius/2019/badge-flair') }}">Badge Flair Contest</a></li>
+
+                                          {% if flaghashike %}
+                                          <li><a href="{{ pathto('conf/portland/2019/outing') }}">Hike</a></li>
```
Comment (plaindocs) [inline docs/_templates/2019/lt/menu-desktop.html]:
```suggestion
                                          <li><a href="{{ pathto('conf/vilnius/2019/outing') }}">Hike</a></li>
```

File: docs/_templates/2019/lt/menu-mobile.html (hunk 1 above)
Comment (thatdocslady) [reply docs/_templates/2019/lt/menu-mobile.html:20]: There won't be a hike AFAIK or another outing, can we remove this entirely?

Comment (plaindocs) [reply docs/_templates/2019/lt/menu-mobile.html:20]: Sure, you probs want to delete lines 19-22 or wherever the condition ends, and `outing.rst` ;-)

**Comment by plaindocs** [comment]
@aidiss can you take a look at the sponsorship prospectus as well, seems to be some hardcoded dates and locations in there that need updating.

File: docs/conf/vilnius/2019/sponsors/prospectus.rst [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+.. raw:: pdf
+
+   PageBreak
+
+.. contents::
+   :local:
+   :depth: 1
+   :backlinks: none
+
+**New this year**: our `Job Fair`_.
```
Comment (thatdocslady) [inline docs/conf/vilnius/2019/sponsors/prospectus.rst]: Remove this

File: docs/conf/vilnius/2019/sponsors/prospectus.rst [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+world, not able to connect with their community. Write the Docs is a
+magical experience for many of our attendees, allowing them to feel like
+they have found their place. We believe this is the most
+important thing that our event can do, and we aim to provide that
+experience again this year, and make it better than ever.
+
+Demographics
+============
+
+We hold two conferences yearly, one in Portland, USA in May, and one in
```
Comment (thatdocslady) [inline docs/conf/vilnius/2019/sponsors/prospectus.rst:42]: This should be updated, maybe something like:

We hold long-term conferences in Portland, USA every May, and in Prague, Czech Republic, every September. This year we are happy to launch a brand new event in Vilnius, Lithuania, in June, with an estimated capacity of 50 attendees. 

Also, maybe link to PDX and PRG events?

File: docs/conf/vilnius/2019/sponsors/prospectus.rst [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+
+   PageBreak
+
+Sponsorship Packages
+====================
+
+The following options are suggested sponsorship levels. We are happy to discuss
+adjustments and custom packages.
+
+These package prices apply to the Portland conference only. Look here for
```
Comment (thatdocslady) [inline docs/conf/vilnius/2019/sponsors/prospectus.rst]: These prices apply to the Vilnius conference only.
(Plus update the links to 2019 Portland and Prague)

Discounted packages for sponsoring all conferences

File: docs/conf/vilnius/2019/sponsors/prospectus.rst [hunk 6]
```diff
@@ -0,0 +1,190 @@
...
+---------
+
+The **Publisher** package is great for sending a team and getting to know the community.
+
+- Five (5) tickets_
+- Large logo & link on the Write the Docs website
+- Name included in welcome announcement in email newsletters and social media
+- Display 2 promotional (“Swag”) item on the conference swag table (provided by sponsor)
+- Two featured job postings in our Newsletter (2,500 subscribers)
+- A table at the job fair
```
Comment (thatdocslady) [inline docs/conf/vilnius/2019/sponsors/prospectus.rst]: No job fair

File: docs/conf/vilnius/2019/sponsors/prospectus.rst [hunk 7]
```diff
@@ -0,0 +1,190 @@
...
+
+- sponsorship@writethedocs.org
+
+.. _ticket: https://ti.to/writethedocs/write-the-docs-portland-2018/
+.. _tickets: https://ti.to/writethedocs/write-the-docs-portland-2018/
+
+Run of Show
+===========
+
+An overview of the conference schedule:
```
Comment (thatdocslady) [inline docs/conf/vilnius/2019/sponsors/prospectus.rst:141]: Need to update to match schedule and venue

File: docs/conf/vilnius/2019/sponsors/prospectus.rst [hunk 8]
```diff
@@ -0,0 +1,190 @@
...
+--------------------------------------------
+
+Come prepared to engage with our community, and to learn just as much as you teach. Engage with our event as attendees as well as sponsors. Send technical staff who can chat with people on the interesting things your company is doing, and get value from the vast amount of insight in the room. We do have some decision makers in the room, but soft sells will work better than hard sales in the environment we strive for.
+
+Who is my primary contact?
+--------------------------
+
+Eric Holscher will be your primary contact, but our team is available at sponsorship@writethedocs.org. If you have a time sensitive inquiry, please email the entire team to ensure a timely response.
+
+What are the dates that sponsors need to know about in advance?
```
Comment (thatdocslady) [inline docs/conf/vilnius/2019/sponsors/prospectus.rst:156]: update schedule

File: docs/conf/vilnius/2019/sponsors/prospectus.rst [hunk 9]
```diff
@@ -0,0 +1,190 @@
...
+------------------------------------------------------------------
+
+* **SUNDAY**: The swag tables are available during the Writing Day, if you want to put out swag items early. Sponsor booths are available for setup on Sunday from 10am-4pm, but don't need people in them yet.
+
+* **MONDAY**: Doors open at 8am, so we recommend arriving around this time to get the most interaction with attendees. This is the official start of the conference, when sponsors tables will need to be staffed. The conference will run until around 5pm.
+
+How do I use get my free tickets?
```
Comment (thatdocslady) [inline docs/conf/vilnius/2019/sponsors/prospectus.rst]: use/get tickets?

File: docs/conf/vilnius/2019/sponsors/prospectus.rst [hunk 10]
```diff
@@ -0,0 +1,190 @@
...
+---------------------------------
+
+You should have recieved a unique URL with a discount code for your free sponsorship tickets. We are happy to send it over again, just ask!
+
+What do I need for the job fair?
```
Comment (thatdocslady) [inline docs/conf/vilnius/2019/sponsors/prospectus.rst]: remove

File: docs/conf/vilnius/2019/tickets.rst [hunk 11]
```diff
@@ -0,0 +1,98 @@
...
+
+**Tickets are sold out!**
+
+{% endif %}
+
+Each ticket includes:
+
+* Entry to all conference events and activities
+* Breakfast, snacks, and lunch on all conference days
+* Welcome Reception and Social Event with light snacks and free drinks
```
Comment (thatdocslady) [inline docs/conf/vilnius/2019/tickets.rst]: maybe remove reception and social event until we know for sure whether they're happening

File: docs/conf/vilnius/2019/tickets.rst [hunk 12]
```diff
@@ -0,0 +1,98 @@
...
+* Wifi throughout the event
+* Meeting lots of fantastic people in a spacious, inviting venue
+
+{% if flaghasshirts %}
+
+.. class:: ticket
+
+**Official Conference Shirts**
```
Comment (thatdocslady) [inline docs/conf/vilnius/2019/tickets.rst:28]: remove this (no shirts or any other swag for first-year unless we get surprised with a ton of sponsorship)

**Review by thatdocslady — CHANGES_REQUESTED** [review]
Looks great, a few specific comments on the prospectus and schedule/events

File: docs/conf/vilnius/2019/tickets.rst (hunk 12 above)
Comment (plaindocs) [reply docs/conf/vilnius/2019/tickets.rst:28]: Presumably we'll do print on demand though right?

**Comment by plaindocs** [comment]
I think with @aidiss and my last two commits this should be publishable. Last set of :eyes: appreciated.

**Review by thatdocslady — APPROVED** [review]
LGTM!

**Pull request merged**